src/game.js

```diff
@@ -0,0 +1,97 @@
+var Frame = require('../src/frame.js');
+var TenthFrame = require('../src/tenthFrame.js');
+var Updater = require('../src/updater.js');
+
+function Game() {
+  this.frames = []
+  this.updater = new Updater();
+  this.increment = 0;
+  this.createFrames();
+}
+
+Game.prototype.addFrame = function(frame){
+  this.frames.push(frame);
+}
+
+Game.prototype.createFrames = function(){
+  for (var i = 0; i < 9; i++) {
+     this.addFrame(new Frame());
+  }
+  this.addFrame(new TenthFrame());
+}
+
+Game.prototype.getFrame = function(frameIndex) {
+  return this.frames[frameIndex];
+}
+
+Game.prototype.getTotal = function() {
+  this.update();
+  var total = 0;
+  this.frames.forEach(function(frame){
+    total += frame.getFinalFrameScore();
+  });
+  return total;
+}
+
+Game.prototype.addFrame = function(frame){
+  this.frames.push(frame);
+}
+
+Game.prototype.update = function() {
+  this.updater.update(this.frames);
+}
+
+Game.prototype.inputScore = function(score){
```

Comment:
I'm sure you know that this is too long for a function. You may want to extract part of it in an other function.
Maybe everything inside the condition `if (this.increment == 9)`

Reply:
Yep, I need to refactor this out - I had trouble getting the logic right so I concentrated on that to begin with and didn't get around to refactoring.